addons/backpacks/anarc164/CfgVehicles.hpp

```diff
@@ -14,5 +14,6 @@ class TFAR_anarc164: TFAR_Bag_Base {
     tf_dialog = "anarc164_radio_dialog";
     tf_subtype = "airborne";
     tf_dialogUpdate = "[""%1""] call TFAR_fnc_updateLRDialogToChannel;";
+    tf_halfDuplexOverride = 0; //0 disables half-duplex override, everything greater than 0 enables it
```

Comment:
Default to 0 should probably be inside TFAR_Bag_Base because that's also what 3rd party mods would inherit